tests/jsunit/horizontal_tests.html

```diff
@@ -7,8 +7,8 @@
     <script>goog.require('goog.testing.jsunit');</script>
   </head>
   <body>
+    <script src="test_utilities.js"></script>
```

Comment:
There's a merge conflict here (since I changed these files in my procedure testing PR) that should be easy to resolve.

Reply:
Fixed a28299d